book/src/sprites.md

```diff
@@ -0,0 +1,16 @@
+# Sprites
+
+Sprites are 2D images that represent an object or background. Sprites are represented by two main chunks of data:
+
+* **Texture:** The image made of pixels.
+* **Sprite Layout:** The (rectangular) coordinates of the sprites on that image.
+
+In Amethyst, these are represented by the [`Texture`][doc_tex] and [`SpriteSheet`][doc_ss] types respectively. The pages in this section will explain how to set up your application to load and display sprites.
+
+> **Note:** To see complete application example code, please refer to the [*sprites*][ex_sprites] or [*sprites_ordered*][ex_ordered] examples in the [examples][ex_all] directory.
```

Comment:
This sentence feels a bit choppy, not sure what to change it to though.

Reply:
Reworded it, hopefully less choppy :crossed_swords: